core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -74,6 +74,8 @@ statement
         RENAME TO to=qualifiedName                                     #renameTable
     | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
         ADD COLUMN (IF NOT EXISTS)? column=columnDefinition            #addColumn
+    | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
```

Comment:
This should be part of the existing “add column” syntax (and everything downstream)

Reply:
@martint It was the initial implementation, but extracted because of https://github.com/trinodb/trino/pull/16321#discussion_r1138841802. Are you suggestion to change `columnDefinition`?